Gems/Atom/Feature/Common/Assets/Shaders/Materials/BasePBR/BasePBR_LightingData.azsli

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+// This #define magic lets you use the LightingData class in this file without making it the final LightingData
+// used in your shader. Simply #define LightingData to your custom definition before including this file
+// 
+#ifndef LightingData
+#define LightingData        LightingData_BasePBR
+#endif
+
+#include <viewsrg.srgi>
+#include <Atom/Features/Debug.azsli>
+#include <Atom/Features/LightCulling/LightCullingTileIterator.azsli>
+#include <Atom/Features/PBR/LightingUtils.azsli>
+
+class LightingData_BasePBR
+{
+    LightCullingTileIterator tileIterator;
+    
+    // Lighting outputs
+    float3 diffuseLighting;
+    float3 specularLighting;
+
+    // Factors for the amount of diffuse and specular lighting applied
+    float3 diffuseResponse;
+   
+    // Shrink (absolute) offset towards the normal opposite direction to ensure correct shadow map projection
+    float shrinkFactor;
```

Comment:
IIRC `shrinkFactor` is not required in `LightingData` since we already grab it from `Surface`.